README.md

```diff
@@ -77,6 +77,7 @@ want to contribute? here is the source code https://github.com/linkedin-faq
 | [Microsoft Azure](microsoft-azure/microsoft-azure-quiz.md)                   | 55        | 53      | [@tomtreffke](https://github.com/tomtreffke), [@ziasistani](https://github.com/ziasistani)                                                           |                                                                                                                                                                                      |
 | [Microsoft Excel](microsoft-excel/microsoft-excel-quiz.md)                   | 109       | 107     | [@gazihasanrahman](https://github.com/gazihasanrahman)                                                                                               | <kbd>[<img title="Deutsch" alt="Deutsch" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/de.svg" width="22">](microsoft-excel/microsoft-excel-quiz-de.md)</kbd> |
 | [Microsoft Outlook](microsoft-outlook/microsoft-outlook-quiz.md)             | 79        | 56      |                                                                                                                                                      |                                                                                                                                                                                      |
+| [Microsoft Power Automate](microsoft-power-automate/microsoft-power-automate-quiz.md)          | 14        | 02      |                                                                                                                                                      |                                                                                                                                                                                      |
```

Comment:
@mervynteo thanks for your PR, you might want to push the file with the quiz or you want to keep it for the future. I am fine to merge if you are going to make in the next PR too. Just please let me know your plan! thanks

Reply:
ok